backend/src/models/api/DefenceConfigureRequest.ts

```diff
@@ -1,11 +1,15 @@
 import { Request } from 'express';
 
+import { ChatInfoMessage } from '@src/models/chatMessage';
 import { DEFENCE_ID, DefenceConfigItem } from '@src/models/defence';
 import { LEVEL_NAMES } from '@src/models/level';
 
 export type DefenceConfigureRequest = Request<
 	never,
-	null | string,
+	| string
+	| {
+			resultingChatInfoMessage: ChatInfoMessage;
```

Comment:
`string` or `object` - is that cos we have some error responses with a string body?

What do we do with those error responses? I'm now checking your changes in defenceService...

Reply:
Yes, sometimes we return an error message a simple string.  Also yes, we swallow those. I suppose they remain helpful because you can see them in the browser inspector.

A note on these request types: This second part (ResBody) seems entirely redundant. TypeScript doesn't seem to enforce the types at all when you call `res.send(...)`. Are we missing something?